src/clipper/ytdl.py

```diff
@@ -147,8 +147,8 @@ def ytdl_lib_get_video_info(cs: ClipperState) -> Tuple[Dict, str]:
         "youtube_include_dash_manifest": False,
     }
 
-    if settings["cookiefile"] != "":
-        ytdl_opts["cookiefile"] = settings["cookiefile"]
+    if settings["cookies"] != "":
+        ytdl_opts["cookies"] = settings["cookies"]
```

Comment:
Shouldn't this be `cookiefile` still? yt-dlp sets `cookiefile` using the `--cookies` option.

We could also just remove these yt-dlp lib-based functions as they aren't being used anymore.

Reply:
Alright, I will remove the lib functions in this PR.